src/controller/DevicePairingDelegate.h

```diff
@@ -70,6 +70,19 @@ class DLL_EXPORT DevicePairingDelegate
      *   Called when the commissioning process is complete (with success or error)
      */
     virtual void OnCommissioningComplete(NodeId deviceId, CHIP_ERROR error) {}
+
+    /**
+     *   Called when the commissioning process is complete (with success or error)
+     *   Should be overriden by DevicePairingDelegates which are JF compatible
+     *   - trustedIcacPublicKeyB has a value only when Joint Commissioning Method is executed
+     */
+    virtual void OnCommissioningComplete(NodeId deviceId,
+                                         chip::Optional<chip::Credentials::P256PublicKeySpan *> trustedIcacPublicKeyBSpan,
```

Comment:
Why is this not taking `const Optional<Credentials::P256PublicKeySpan> &` or `const Optional<Crypto::P256PublicKey> &` so?

(Note the type, and the lack of redundant `chip::` prefixes).

Reply:
Now it does. Please check.